e2e-tests/desktop/pages/sessionPage.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import GlobalPage from './globalPage.js';
+
+export default class SessionPage {
```

Comment:
Should this have extended the `globalPage` rather than importing here? We also called it `basePage` in admin, should they be called the same?